src/client/features/interactions/index.js

```diff
@@ -136,9 +137,42 @@ class Interactions extends React.Component {
     });
   }
 
+  generateCentralityValues(){
+    const state = this.state;
+    const cy = state.cy;
+    const bc = cy.$().bc();
+    const nodes = cy.nodes();
+    if(nodes.length === 0) return;
+    
+    let centralityVals = [];
+    let centralityMap = [];
+    nodes.forEach( (ele) => {
+      if(ele.data){
+        let bcVal = bc.betweenness(ele);
+        centralityVals.push(bcVal);
+        centralityMap.push([ele,bcVal]);
+      }
+    });
+
+    const max = Math.max(...centralityVals);
+    const min = Math.min(...centralityVals);
+
+
+    centralityMap.forEach( (ele) => {
+      ele[0].data('bcVal',this.normalizeValue(ele[1],max,min));
+    });
+
+  }
+
+
+  normalizeValue(value,max,min){
+    return (value-min)/(max-min);
+  }
+
   render(){
     const state = this.state;
     const loaded = state.loaded;
+    this.generateCentralityValues();
```

Comment:
I feel like this is the wrong place to run this function, since it looks like it's getting called a bunch of times with an empty network, but I'm not sure where to put it otherwise...